### PR TITLE
Add travis scripts

### PR DESCRIPTION
**Pull request by andersonfrailey**
Add yml file to run travis.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  - "3.7"
+
+install:
+  - wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - export PATH="$HOME/miniconda/bin:$PATH"
+  - conda config --set always_yes yes
+  - conda update conda
+  - conda create -n taxbrain-dev python=$TRAVIS_PYTHON_VERSION;
+  - source activate taxbrain-dev
```
Comment (hdoupe) [inline .travis.yml]: What happens if you swap this to `conda activate taxbrain-dev`?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  - "3.7"
+
+install:
+  - wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - export PATH="$HOME/miniconda/bin:$PATH"
+  - conda config --set always_yes yes
+  - conda update conda
+  - conda create -n taxbrain-dev python=$TRAVIS_PYTHON_VERSION;
+  - conda activate taxbrain-dev
```
Comment (hdoupe) [inline .travis.yml]: Maybe add `conda init bash` before `conda activate-taxbrain-dev`?

Comment (hdoupe) [reply .travis.yml]: Ah, nevermind. It's hopeless. Maybe go back to `source activate taxbrain-dev`?

**Comment by hdoupe** [comment]
@andersonfrailey this seems like a memory error (https://github.com/travis-ci/travis-ci/issues/4192). Are the tests running code that uses multiple processes? Tax-Brain uses a bunch of memory since it has one or more Tax-Calculator `Calculator` objects in each process. That could be the issue.

**Comment by andersonfrailey** [comment]
@hdoupe ah that would make sense. I might be able to configure which tests run and make it work. Thanks for helping with this.

**Comment by hdoupe** [comment]
No problem, you may be able to limit dask to only 1 worker, too.

**Comment by andersonfrailey** [comment]
Closing since this PR is outdated and automated testing will be added in PR #160

**Pull request closed without merging**